experimenter/experimenter/experiments/constants.py

```diff
@@ -18,6 +19,26 @@ class Channel(models.TextChoices):
     DEVELOPER = "developer"
 
 
+class ChangeEventType(Enum):
+    GENERAL = "GENERAL"
+    CREATION = "CREATION"
+    DETAILED = "DETAILED"
+    STATE = "STATE"
+    BOOLEAN = "BOOLEAN"
+
+
+class RelationalFields:
```

Comment:
This needs documentation.

Also, why is this a class?

Reply:
I just tried to declare constants similar to how I did for events, is this  incorrect ?